src/components/Facility/PatientNotesSlideover.tsx

```diff
@@ -153,15 +153,16 @@ export default function PatientNotesSlideover(props: PatientNotesProps) {
         id="expand_doctor_notes"
         className={classNames(
           "tooltip flex h-8 w-8 cursor-pointer items-center justify-center rounded bg-primary-800 text-secondary-100 text-opacity-70 hover:bg-primary-700 hover:text-opacity-100",
-          show && "rotate-180",
         )}
         onClick={() => setShow(!show)}
       >
         <CareIcon
-          icon="l-angle-up"
+          icon={show ? "l-angle-down" : "l-angle-up"}
           className="tooltip text-lg transition-all delay-150 duration-300 ease-out"
         />
-        <span className="tooltip-text tooltip-top rotate-[-180deg] text-xs">
+        <span
+          className={`tooltip-text tooltip-${show ? "bottom -translate-x-16" : "top"} text-xs`}
+        >
```

Comment:
use cn or classNames utils to add conditional classes, string interpolation can be hard to read sometimes

Reply:
@khavinshankar ,Done.